application.tf

```diff
@@ -1,12 +1,91 @@
 ####################################################
-###...Apply The 2048 game in Kubernetes Cluster
-###...to verify AWS ALB ingress controller work
+###    Game-2048 Deployment                      ###
+### For Test ingress controller                  ###
 ####################################################
 
-resource "null_resource" "install_2048" {
-  provisioner "local-exec" {
-    command = "kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.1.3/docs/examples/2048/2048_full.yaml"
-  }
+resource "kubectl_manifest" "game-2048-namespace" {
+  yaml_body = <<YAML
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: game-2048
+YAML
 
+  wait       = true
   depends_on = [helm_release.aws_ingress]
+}
+
+resource "kubectl_manifest" "game-2048-deployment" {
+  yaml_body =<<YAML
+apiVersion: apps/v1
```

Comment:
is it possible to expose yaml to standalone yaml file?

Reply:
Fixed in next PR.